src/singletons/Settings.hpp

```diff
@@ -316,6 +316,8 @@ class Settings : public ABSettings, public ConcurrentSettings
         "/misc/attachExtensionToAnyProcess", false};
     BoolSetting hideViewerCountAndDuration = {
         "/misc/hideViewerCountAndDuration", false};
+    BoolSetting hideImagesInStreamerMode = {"/misc/hideImagesInStreamerMode",
```

Comment:
something like `/misc/streamerMode/hideUserContent` would maybe make more sense?

Reply:
maybe even `/streamerMode/hideUserContent` might be ok

Reply:
>
> maybe even /streamerMode/hideUserContent might be ok

I went ahead and applied this suggestion in b9503b7, however split link thumbnails and usercard avatars to two separate settings as alazymeme requested in his issue.